tests/test_main.py

```diff
@@ -164,4 +164,4 @@ def test_unnecessary_comment():
 
         assert (
                 "# write your code here" not in main_content
-        ), "Remove unnecessary comment"
+        )
```

Comment:
It seems that you face difficulties with removing this file from the PR since it was asked several times to do that. Please, visit Q&A to make it together with your mentor and resolve this task. Try again to do that beforehand, this file should be absent in the PR

Reply:
do it